Backend/offers/offersController.js

```diff
@@ -0,0 +1,112 @@
+const offersModel = require("./offersModel.js");
+const { updateBody } = require("./offersHelper");
+const io = require("../socket");
+const { ResponseErrorHandler } = require("../ErrorsHandler/ResponseErrorHandler.js");
+const { tryCatchHandler } = require("../ErrorsHandler/TryCatchHandler.js");
+
+module.exports.getOffersByMakerId = async (req, res) => {
+
+  const tryFn = async () => {
+
+    const maker_id = req.params;
+
+    const myOffers = await offersModel.fetchMyOffers(maker_id);
+
+    return res.status(200).json(myOffers);
+  }
+  return tryCatchHandler(tryFn)
+};
+
+module.exports.getAllOffers = async (req, res) => {
+
+  const tryFn = async () => {
+
+    const allOffers = await offersModel.fetchAllOffers();
+
+    return res.status(200).json(allOffers);
+  }
+  return tryCatchHandler(tryFn)
+};
+
+module.exports.getOffer = async (req, res) => {
+
+  const tryFn = async () => {
+    const { id } = req.params;
+
+    if (!id) {
+      ResponseErrorHandler(res, 400, "Please provide a valid id")
```

Comment:
400 does not look like a error for me. 500x

I think we can just res.send() here

Reply:
ok

Reply:
Check